Tiltfile.starter

```diff
@@ -0,0 +1,31 @@
+# Welcome to Tilt!
+# To get you started as quickly as possible, we have created this generic Tiltfile. 
+# Modify and comment out any commands to make them work for you.
+# Happy Tilting!
+
+
+# use a custom remote registry
+# more info on https://docs.tilt.dev/personal_registry.html
+# default_registry('gcr.io/my-public-registry')
+
+# run a one-time command e.g. to customize your cluster
+# more info on https://docs.tilt.dev/local_resource.html
+# local_resource('my-command', cmd='./scripts/my-command.sh', deps=['scripts/my-command.sh'], allow_parallel=True, auto_init=False)
```

Comment:
I was trying to think of useful cross-platform local commands to run, but I'm coming up empty :). I think these are good for now.

Reply:
❓ 
```suggestion
local_resource('hello-local', cmd='echo Hello World')
```

Reply:
One idea I had: install some cluster tooling that is used on prod, and you want to create the same conditions on your local cluster, for example OPA Gatekeeper